prog/vm/nexus.rb

```diff
@@ -34,11 +41,21 @@ def self.assemble(public_key, project_id, name: nil, size: "m5a.2x",
       private_subnets.each do
         VmPrivateSubnet.create(vm_id: vm.id, private_subnet: _1.to_s)
       end
+
+      if storage_encrypted
```

Comment:
Do we even want a branch here? How about always?

Reply:
answer: in non-storage (often network-related) scenarios, encryption can make debugging really painful unless we see a resolution to https://spdk-team.slack.com/archives/CJE4C98G1/p1688495434441229, which reads:

```
I have a development-process related question, from the point of view of an SPDK user: so, we're using SPDK to make block devices available to virtual machines via vhost-user, as one does, and we're adding more features to the storage stack, such as encryption. All good, working as expected.
The development process issue occurs when we're making changes in unrelated areas that can break cloud-init and early boot processes, e.g. when working on networking in a way that can break SSH. In the past, we'd do stuff like mount the raw image found as a loopback device and twiddle a few files or look at things, bypassing SPDK. But as the features we're getting out of SPDK grows, it gets harder to do this, e.g. once the blocks are encrypted. So we need to find a way to get access to the block device on the host while continuing to let SPDK do its work.
In addition, SPDK is running in non-root mode, so some approaches, like nbm, are awkward, and require a kernel module not available by default, which is a bit dissatisfying. Ditto nvme-of tcp. and ublock requires a very new kernel.
We could, in principle, start another VM that talks via vhost-user to that block device, but that's pretty brutal to work with in a development situation.
The arrangement that seems most direct to me, but also kind of weird, would to be able to have Linux, on the host, talk to vhost-user directly as though it were a VM, and expose that as a block device. Is that something anyone has done? Or is there some other way to accomplish this?
```

Reply:
I kept it as is for now to make development debugging easier. I'll revisit this later.